### PR TITLE
Flip semantics of ELIST C and ELIST T

### PR DESCRIPTION
**Pull request by progval**
The spec <https://datatracker.ietf.org/doc/html/draft-hardy-irc-isupport-00#section-4.8>
is a little unclear on the semantics, and Unreal interprets 'C<val' as
a request for all chans created before <val> minutes ago, ditto.

This is a legitimate interpretation, but I think the other on makes more
sense (ie. that 'C<val' means all chans created less than <val> minutes
ago).

Additionally, the documentation for T did not match the implementation
before this commit.

Before this commit, Unreal was consistent with Hybrid/Plexus4 on
ELIST C.

After this commit, Unreal will be consistent with Charybdis/Solanum,
InspIRCd, and ircu2 (and Hybrid/Plexus4 on ELIST T).

### CODE REVIEW

**Comment by syzop** [comment]
If this is the consensus then my initial reaction is: this change sounds fine to me.
If I read that (old) draft with no prior knowledge then I would interpret it as a unix timestamp argument (absolute time since epoch), but clearly it has been implemented completely differently in practice (everywhere?) to mean something in "relative time"... in minutes even... so that draft is not of any help then.

How recent were the changes in InspIRCd (if at all changed) for "ELIST C"? Do major clients (eg: mIRC) use it one way or the other?
In other words: how much chance has this change of screwing up some existing client? ;)

**Comment by progval** [comment]
> How recent were the changes in InspIRCd (if at all changed) for "ELIST C"?

According to the Git history, it was added in 2017 and not changed.

> Do major clients (eg: mIRC) use it one way or the other?

I don't think so, but I'll look into it

**Comment by progval** [comment]
I can't find a single client that even uses it. Couldn't check mIRC, though.

@ValwareIRC Any idea?

**Comment by progval** [comment]
```
<Valware> mIRC never requests any list without the user asking
```

Looks like we're good, then

**Comment by syzop** [comment]
Ok, thanks! :)
